apps/vm/fsw/src/vm_app.cpp

```diff
@@ -840,7 +825,6 @@ void VM::ProcessAppCmds(CFE_SB_Msg_t* MsgPtr)
                 HkTlm.SensorAccelMsgCount             = 0;
                 HkTlm.SafetyMsgCount                  = 0;
                 HkTlm.SensorCorrectionMsgCount        = 0;
-                HkTlm.VehicleStatusMsgCount           = 0;
```

Comment:
Are we still subscribing to this message anywhere?  If so, remove the subscription.

Reply:
This PR also removes the subscription as VM is the sender and doesn't need to receive this message.